step1-clean/db.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine, Column, String, Integer
+
+Base = declarative_base()
+
+class SQLImage(Base):
+  __tablename__ = 'Thumbnails'
+
+  id = Column(Integer, primary_key=True)
+  filename = Column(String)
+  isThumbnail = Column(String)
+
+  def __repr__(self):
+    return "SQLImage<id={id}, filename='{filename}' isThumbnail='{isThumbnail}'>".format(
+      id=self.id,
+      filename=self.filename,
+      isThumbnail=self.isThumbnail
+    )
+
+
+engine = create_engine('sqlite+pysqlite:///GG.db')
```

Comment:
In python, code runs every time it's imported it as a module (`from db import SQLImage`). This means the database will be configured every time the mode is imported.

Please separate the Image model class to a models.py

Reply:
I've moved the model classes to models.py. I partially understand what you are saying, but would like some clarity on code running when modules imported.

Reply:
![lol](https://user-images.githubusercontent.com/1236436/77184159-cbd36b80-6aa5-11ea-9c53-2b84a9a4f2ff.png)

Reply:
Got it!